pending_changelog/monitorupdatingpersister.txt

```diff
@@ -0,0 +1,6 @@
+## Backwards Compatibility
+
+* Monitors stored with [`MonitorUpdatingPersister`] have a prepended sentinel value that prevents them from being deserialized by previous `Persist` implemtentions. This is to ensure that they are not accidentally read and used while pending updates are still stored separately, as this could result in penalty transactions. Users who wish to downgrade should perform the following steps:
+  * Make a backup copy of all channel state.
+  * Ensure all updates are applied to the monitors. This may require starting the node with 0.0.117 and immediately performing an orderly shutdown, or running a small crafted utility program. Restricting your node's network access during this process is a good precaution.
```

Comment:
> starting the node with 0.0.117 and immediately performing an orderly shutdown

Is this correct? Do full CMs get persisted on an orderly shutdown?

Reply:
Hm, so that doesn't really work as a downgrade path. You'd need just a little more code; you have to call `read_all_channel_monitors_with_updates`, and then map `persist_new_channel` over those. There are some additional arguments you might need to put together. Perhaps I should include a `consolidate_channels` function that does this, to give a downgrade path?